scripts/update-precache-blocks.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
```

Comment:
this script will be executed by the devops in their infra?

Reply:
Yes, probably it will be executed inside the cache docker container

Reply:
Now, I am thinking about defining another strategy for deployment from scratch. There we should define starting blocks as blocks on which contract was deployed